src/units/abilities.hpp

```diff
@@ -18,42 +18,3 @@
 #include "map/location.hpp"
 #include "units/ptr.hpp"
 
-class unit_ability_list;
```

Comment:
Having both "ability.cpp" and "abilities.cpp" feels like it'll be an source of annoyance when forgetting which one is which. Is there a plan to have only one by the end of the refactor?

Leaving an empty file called "abilities.hpp" while adding "ability.hpp" also seems bad.

I understand the desire to split what's now in ability.cpp from the mish-mash of stuff in abilities.cpp, however I think it's the mish-mash that should move instead.

Reply:
> Is there a plan to have only one by the end of the refactor?

I was hoping so, not sure yet whether ill actually get there.